.travis.yml

```diff
@@ -11,26 +11,34 @@ matrix:
   fast_finish: true
   include:
     - php: 5.6
-      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.0
       env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1 PHPCS_TEST=1
     - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1 PDO=1
+    - php: 7.3
+      env: DB=MYSQL PHPUNIT_TEST=1
+    - php: 7.3
       env: DB=PGSQL PHPUNIT_TEST=1
 
 before_script:
+# Init PHP
   - printf "\n" | pecl install imagick
   - composer self-update || true
   - phpenv rehash
-  - phpenv config-rm xdebug.ini
```

Comment:
This is deliberate - xdebug slows phpunit down a LOT, and we use phpdbg for code coverage runs instead in PHP 7

Reply:
that's actually been the fix I had to do to fix coverage generation in the branch `4`. not sure it's gonna work without it, but can check it in another branch

Reply:
PHP 7 has phpdbg bundled with it, which is why we usually run coverage builds on PHP 7 - most core and supported modules already have this config in place if you need a reference

Reply:
ok, it's working on this branch without xdebug. Will put it back then and we'll keep an eye on codecov when merge it up

Reply:
> that's actually been the fix I had to do to fix coverage generation in the branch 4

yeah, my bad, I'm clumsy today...